# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../device-mapper/dietlibc.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./dmsetup/dmsetup.c.orig	2007-08-15 16:01:23.000000000 -0400
+++ ./dmsetup/dmsetup.c	2007-08-15 16:01:45.000000000 -0400
@@ -37,6 +37,7 @@
 #include <sys/param.h>
 #include <locale.h>
 #include <langinfo.h>
+#include <stdint.h>
 
 #include <fcntl.h>
 #include <sys/stat.h>
